ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/adapter/OptInOnBoardingPagerAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package org.cru.godtools.tutorial.adapter
+
+import androidx.annotation.StringRes
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import org.cru.godtools.tutorial.R
+import org.cru.godtools.tutorial.fragment.OptInOnBoardingSlideFragment
+import org.cru.godtools.tutorial.util.OnBoardingCallbacks
+
+class OptInOnBoardingPagerAdapter(val callbacks: OnBoardingCallbacks, fm: FragmentManager) :
+    FragmentPagerAdapter(fm, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
```

Comment:
There is no need to make these fragments since you are just using data binding with them, you should utilize `DataBindingPagerAdapter` instead.

https://github.com/CruGlobal/android-gto-support/blob/master/gto-support-viewpager/src/main/java/org/ccci/gto/android/common/viewpager/adapter/DataBindingPagerAdapter.kt

Reply:
@frett I am using different ViewDataBinding so I don't see how this will work for me.  I am going to try to remove the fragment and just use a basic PageAdapter (to simplify things).